wagtail_wordpress_import/test/fixtures/raw_xml.xml

```diff
@@ -252,6 +252,35 @@ Nihil hic munitissimus habendi senatus locus, nihil horum?.&lt;/blockquote&gt;</
                 <wp:meta_value>a search description from yoast using a different key</wp:meta_value>
             </wp:postmeta>
         </item>
+        <item>
+            <title>Item with wp:postmeta</title>
+            <link>https://www.example.com/item-one-title/</link>
+            <pubDate>Tue, 13 Jul 2010 16:16:46 +0000</pubDate>
+            <dc:creator>author1</dc:creator>
+            <guid isPermaLink="false">https://www.example.com/wp-content/uploads/2010/07/haunted-house.jpg</guid>
+            <description>This page has a default description</description>
+            <content:encoded></content:encoded>
+            <excerpt:encoded>Lorem ipsum dolor sit amet</excerpt:encoded>
+            <wp:post_id>1</wp:post_id>
+            <wp:post_date>2010-07-13 12:16:46</wp:post_date>
+            <wp:post_date_gmt>2010-07-13 16:16:46</wp:post_date_gmt>
+            <wp:post_modified>2010-07-13 12:16:46</wp:post_modified>
+            <wp:post_modified_gmt>2010-07-13 16:16:46</wp:post_modified_gmt>
+            <wp:comment_status>open</wp:comment_status>
+            <wp:ping_status>open</wp:ping_status>
+            <wp:post_name>item-one-title</wp:post_name>
+            <wp:status>publish</wp:status>
+            <wp:post_parent>0</wp:post_parent>
+            <wp:menu_order>0</wp:menu_order>
+            <wp:post_type>postmetasingle</wp:post_type>
+            <wp:post_password/>
+            <wp:is_sticky>0</wp:is_sticky>
+            <category domain="category" nicename="blogging">Blogging</category>
+            <wp:postmeta>
+                <wp:meta_key>_thumbnail_id</wp:meta_key>
+                <wp:meta_value><![CDATA[43124]]></wp:meta_value>
+            </wp:postmeta>
+        </item>
```

Comment:
**Question**: What role does this new fixture have? All the unit tests in https://github.com/torchbox/wagtail-wordpress-import/pull/72/files#diff-87e9fedd3c41cadd0bc524052e05b7d963c00dbd029ac32a3ef85757b3e24974R214 seem to use the `post_meta.xml` file.

Reply:
Thanks Nick. That fixture is used in [test_wordpress](https://github.com/torchbox/wagtail-wordpress-import/blob/fix/%2380-yoast-import-error/wagtail_wordpress_import/test/tests/test_wordpress_importer.py) The fixture didn't have the single wp_post_meta tag to test.

I think I added it there before deciding to do the tests in test_wordpress_item.py. I'll have a look but rather than remove the fixture it i'll add a new test for it in test_wordpress.py

Reply:
Actually doesn't seem like test_wordpress.py is a good place for a test like that as we don't do anything with _thumbnail_id. We will in the next piece of work but not here. I'll remove it.

Reply:
I reverted this extra fixture and the commit where the total need to be updated.